src/Cli/dotnet/Parser.cs

```diff
@@ -53,18 +53,31 @@ public static class Parser
             HelpCommandParser.GetCommand(),
             SdkCommandParser.GetCommand(),
             InstallSuccessCommand,
-            WorkloadCommandParser.GetCommand()
+            WorkloadCommandParser.GetCommand(),
+            new System.CommandLine.StaticCompletions.CompletionsCommand()
         };
 
         public static readonly CliOption<bool> DiagOption = CommonOptionsFactory.CreateDiagnosticsOption(recursive: false);
 
-        public static readonly CliOption<bool> VersionOption = new("--version");
+        public static readonly CliOption<bool> VersionOption = new("--version")
+        {
+            Arity = ArgumentArity.Zero,
+        };
 
-        public static readonly CliOption<bool> InfoOption = new("--info");
+        public static readonly CliOption<bool> InfoOption = new("--info")
+        {
+            Arity = ArgumentArity.Zero,
+        };
 
-        public static readonly CliOption<bool> ListSdksOption = new("--list-sdks");
+        public static readonly CliOption<bool> ListSdksOption = new("--list-sdks")
+        {
+            Arity = ArgumentArity.Zero,
+        };
 
-        public static readonly CliOption<bool> ListRuntimesOption = new("--list-runtimes");
+        public static readonly CliOption<bool> ListRuntimesOption = new("--list-runtimes")
+        {
+            Arity = ArgumentArity.Zero,
+        };
```

Comment:
Really these look like options but are more like commands - I'd like to model them that way in the future but previous PRs have broken and I haven't gone back to look at it. In any case, these aren't every actually invoked by us (they are handled by the runtime), they just exist here to correctly model our CLI for help/argument parsing purposes.